src/ResourceManager/Network/Commands.Network/ChangeLog.md

```diff
@@ -19,6 +19,8 @@
 --->
 
 ## Current Release
+* Added WhatIf support to Get-AzureRmNetworkSecurityGroup
```

Comment:
this is not required. Should Process is only required for cmdlets that have server side effects. Gets do not do that and hence are not candidates for shouldprocess. Kindly revert this.